Tests/configure_and_test_integration_instances.py

```diff
@@ -499,6 +521,9 @@ def set_integration_params(integrations, secret_params, instance_names, placehol
             integration['byoi'] = matched_integration_params.get('byoi', True)
             integration['instance_name'] = matched_integration_params.get('instance_name', integration['name'])
             integration['validate_test'] = matched_integration_params.get('validate_test', True)
+            if integration['name'] not in build.unmockable_integrations:
+                integration['params'].update({'proxy': True})
```

Comment:
We should probably also add the converse.
```suggestion
            if integration['name'] not in build.unmockable_integrations:
                integration['params'].update({'proxy': True})
            else:
                integration['params'].update({'proxy': False})
```